Crowd Strike Falcon X - bug with uploading file (#11675)
demisto/content: Packs/CrowdStrikeFalconX/ReleaseNotes/1_1_1.md

#### Integrations
##### CrowdStrike Falcon X

> - Fix an issue with where *cs-fx-upload-file* didn't uploaded exe and dll files correctly.

Reviewer: ```suggestion
- Fix an issue where **cs-fx-upload-file** failed to upload .exe and .dll files correctly.
```

> - Fix an issue where **cs-fx-upload-file** failed to upload .exe and .dll files correctly.

Reviewer 2: ```suggestion
- Fix an issue where ***cs-fx-upload-file*** failed to upload `.exe` and `.dll` files correctly.
```
